src/Analyzers/CSharp/Tests/RemoveUnusedParametersAndValues/RemoveUnusedValueExpressionStatementTests.cs

```diff
@@ -446,8 +446,8 @@ public C()
 
     void M(int unused1, int unused2)
     {{
-        {fix2} = M2();
```

Comment:
@mavasani  Since we sort the diagnostic in descending order this test fails. I am guessing it doesn't matter so I change this. But tag you in case we should maintain the old order.

Reply:
I think this is fine.